src/util/campusUnitConversion.ts

```diff
@@ -0,0 +1,11 @@
+export const celsiusToFahrenheit = (celsius) => {
```

Comment:
Since these are not ONLY for campus stuff we need to make the name of the utility much more generic and describe whats it for ( unit Conversions ) is good but we could also look at a library to use for the first 2.

Reply:
commit ID: e24aa642a274f7ab6cd0679e007abe43357699bb

Addressed obvious file renaming and corrected reference; fully understand desire to use a library for generic conversions, would like instruction how how to implement